packages/prism-theme-odyssey/src/index.js

```diff
@@ -1,3 +1,30 @@
 import '@okta/odyssey';
-import 'prismjs';
+
+import "prismjs";
+import "prismjs/components/prism-csharp";
+import "prismjs/components/prism-clike";
+import "prismjs/components/prism-c";
+import "prismjs/components/prism-go";
+import "prismjs/components/prism-http";
+import "prismjs/components/prism-java";
+import "prismjs/components/prism-markdown";
+import "prismjs/components/prism-python";
+import "prismjs/components/prism-objectivec";
+import 'prismjs/components/prism-markup-templating';
+import "prismjs/components/prism-php";
+import "prismjs/components/prism-swift";
+
+import './demo.scss';
 import './theme.scss';
+
+import CustomPropertyInspector from './utils/CustomPropertyInspector.js'
+
+
+const urlParams = new URLSearchParams(window.location.search);
+const showInspector = urlParams.get('inspector');
```

Comment:
consider using [`has` ](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/has)to avoid the intermediate variable assignment which is only used for the conditional check below

Reply:
I like it. Will fix.